engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -33,36 +34,14 @@
 import org.terasology.rendering.assets.shader.ShaderProgramFeature;
 import org.terasology.rendering.backdrop.BackdropProvider;
 import org.terasology.rendering.cameras.Camera;
-import org.terasology.rendering.cameras.OculusStereoCamera;
+import org.terasology.rendering.cameras.OpenVRStereoCamera;
 import org.terasology.rendering.cameras.PerspectiveCamera;
 import org.terasology.rendering.dag.Node;
 import org.terasology.rendering.dag.NodeFactory;
 import org.terasology.rendering.dag.RenderGraph;
 import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.RenderTaskListGenerator;
-import org.terasology.rendering.dag.nodes.AmbientOcclusionPassesNode;
-import org.terasology.rendering.dag.nodes.BackdropNode;
-import org.terasology.rendering.dag.nodes.BloomPassesNode;
-import org.terasology.rendering.dag.nodes.BlurPassesNode;
-import org.terasology.rendering.dag.nodes.ChunksAlphaRejectNode;
-import org.terasology.rendering.dag.nodes.ChunksOpaqueNode;
-import org.terasology.rendering.dag.nodes.ChunksRefractiveReflectiveNode;
-import org.terasology.rendering.dag.nodes.DirectionalLightsNode;
-import org.terasology.rendering.dag.nodes.DownSampleSceneAndUpdateExposureNode;
-import org.terasology.rendering.dag.nodes.FinalPostProcessingNode;
-import org.terasology.rendering.dag.nodes.FirstPersonViewNode;
-import org.terasology.rendering.dag.nodes.InitialPostProcessingNode;
-import org.terasology.rendering.dag.nodes.LightGeometryNode;
-import org.terasology.rendering.dag.nodes.LightShaftsNode;
-import org.terasology.rendering.dag.nodes.ObjectsOpaqueNode;
-import org.terasology.rendering.dag.nodes.OutlineNode;
-import org.terasology.rendering.dag.nodes.OverlaysNode;
-import org.terasology.rendering.dag.nodes.PrePostCompositeNode;
-import org.terasology.rendering.dag.nodes.ShadowMapNode;
-import org.terasology.rendering.dag.nodes.SimpleBlendMaterialsNode;
-import org.terasology.rendering.dag.nodes.SkyBandsNode;
-import org.terasology.rendering.dag.nodes.ToneMappingNode;
-import org.terasology.rendering.dag.nodes.WorldReflectionNode;
+import org.terasology.rendering.dag.nodes.*;
```

Comment:
This probably got "optimized" by IDE settings thinking wildcards are good after a certain point. If using IntelliJ In Settings -> Editor -> Code Style -> Java -> Imports tab you should be able to set "Class count to use import with '*' to a really high number to avoid that.